src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -963,6 +1229,51 @@ static SequenceComparison createSequenceComparison(final VariantContext variant,
         return sequenceComparison;
     }
 
+    /**
+     * Calculates the fraction of Guanine and Cytosine bases in a window of a given size around a variant.
+     * Note: Since Guanine and Cytosine are complementary bases, strandedness makes no difference.
+     * @param referenceContext The {@link ReferenceContext} for a variant.  Assumed to already be centered on the variant of interest.  Must not be {@code null}.
+     * @param windowSize The number of bases to the left and right of the given {@code variant} to calculate the GC Content.  Must be >=1.
+     * @return The fraction of Guanine and Cytosine bases / total bases in a window of size {@code windowSize} around a variant.
+     */
+    public static double calculateGcContent( final ReferenceContext referenceContext,
```

Comment:
Why is this in a factory class and not its own datasource?

Reply:
@LeeTL1220 Does GC Content need to be a separate annotation?

Reply:
GC content does not have to be separate from GENCODE annotations, no, I guess not.  BUT it does have to be present for all variants.  And I would argue that since you don't really need the transcripts for it (GC content is based on the reference, not the transcript), you would normally not need GENCODE for it, per se.

I'm okay if you want to file an issue (for decoupling GCcontent and ref context from GENCODE), since I think it will be low priority.  Most users are using the GENCODE datasource.

Reply:
Ah.  Ok, sounds good.

Filed as issue #3969